### ISSUE TITLE
4.9 Chancery production bonus per allies don't work

### ISSUE THREAD

**Issue by brsoft1981** [LoneGazebo/Community-Patch-DLL#11022]
_1. Mod Version (X.Y.Z). 4.9 Vox Pupuli with EUI

_2. DirectX Version_

_3. List of Other Mods_: None

_4. Describe the Issue_
Chancery production bonus per allies don't work. I'm attaching a save game just before building the Chancery in my capital.

______________________________________________________________________________________
_5. Save Game From 1 Turn Before (ALWAYS ATTACH THIS IF POSSIBLE)_

[Morocco Save Game.zip](https://github.com/user-attachments/files/15935931/Morocco.Save.Game.zip)

_6. Logs (ALWAYS ATTACH THESE IF POSSIBLE)_

_7. CvMiniDump.dmp File (ATTACH IF REPORTING A GAME CRASH)_

_8. Steps to reproduce the Issue (Optional)_

_9. Screenshots of the Issue (Optional)_

**Comment by azum4roll** [comment]
https://github.com/LoneGazebo/Community-Patch-DLL/commit/c5ca38b59549086a360daa681c7ac64252251477#diff-d5f21ce0c50b76644708883c843d34664bea5e094699d379687813a5efdddfbf

This commit removed the functionality.

**Comment by Hokath** [comment]
#11020 @RecursiveVision I will take my apology in writing :P

**Comment by ilteroi** [comment]
confused ... is this a database issue now or not?

anyway city state yields should go through CvCity::SetBaseYieldRateFromCSAlliance() and CvCity::SetBaseYieldRateFromCSFriendship(). it's just that CvCity::DoTurn() is not the right place.

**Comment by Hokath** [comment]
Not database apparently. 
I've now reinstalled and I can see the table again.

**Comment by axatin** [comment]
@ilteroi You removed the yields given through `GetYieldPerAlly()` from `CvCity::SetBaseYieldRateFromCSAlliance()` and didn't add them anywhere else

**Comment by ilteroi** [comment]
not true. see void CvCity::UpdateYieldsFromExistingFriendsAndAllies(bool bRemove)

i don't see the connection to chanceries right now, chanceries are buildings not traits
